src/components/DateInputField.jsx

```
@@ -0,0 +1,59 @@
+
+      'border-cornflower-500': filled && !error,
+      'border-green-500': isSuccess,
+
+      'bg-gray-100 text-gray-400 cursor-not-allowed border-gray-300': disabled,
+      'border-red-400 placeholder-red-400': error,
+    }
+  );
+
+  return (
+    <div className="w-full space-y-1">
+      {label && <label className="caption-bold">{label}</label>}
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)